vm_callinfo.h

```diff
@@ -28,6 +29,7 @@ enum vm_call_flag_bits {
 #define VM_CALL_SUPER           (0x01 << VM_CALL_SUPER_bit)
 #define VM_CALL_ZSUPER          (0x01 << VM_CALL_ZSUPER_bit)
 #define VM_CALL_OPT_SEND        (0x01 << VM_CALL_OPT_SEND_bit)
+#define VM_CALL_KW_SPLAT_MUT    (0x01 << VM_CALL_KW_SPLAT_MUT_bit)
```

Comment:
`CI_EMBED_FLAG_bits` should be the number of flags.
The total of `CI_..._FLAG_bits` should be bit size of VALUE.